index.html

```diff
@@ -4,7 +4,17 @@
 		<meta charset="UTF-8" />
 		<link rel="icon" type="image/svg+xml" href="/vite.svg" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-		<title>Vite + React + TS</title>
+		<meta name="twitter:card" content="summary_large_image" />
+		<meta name="twitter:title" content="CommunityFocus" />
+		<meta name="twitter:description" content="Create a sharable timer for your team or friends!" />
+		<meta name="twitter:image" content="https://i.imgur.com/gPkVD1Q.png" />
+
+		<meta property="og:title" content="CommunityFocus" />
+		<meta property="og:description" content="Create a sharable timer for your team or friends!" />
+		<meta property="og:image" content="https://i.imgur.com/gPkVD1Q.png" />
+		<meta property="og:url" content="https://www.CommunityFocus.app" />
+		<meta property="og:type" content="website" />
```

Comment:
when in a room like https://deploy-preview-32--communityfocus.netlify.app/tender-brainy-cricket, discord (and possibly other services), don't show the preview. 

Can we make it so that when a user shares a link when in a room, it shows the preview.

For the title, it can say `Join tender-brainy-cricket's shared timer`
For the description, it can say `Create a sharable timer for your team or friends!`

Something like this would be great

![image](https://github.com/CommunityFocus/cf-frontend/assets/45009203/dd8e2918-e7ca-46aa-9b99-3b0310b8a474)

Reply:
Would you also be able to make the icon smaller like the above screenshot?

Currently, it shows up much bigger like

![image](https://github.com/CommunityFocus/cf-frontend/assets/45009203/228ad425-f50e-46ef-8a82-ca79fc55bed0)